cmd/csaf_provider/controller.go

```diff
@@ -69,15 +70,24 @@ func (c *controller) bind(pim *pathInfoMux) {
 func (c *controller) auth(
 	fn func(http.ResponseWriter, *http.Request),
 ) func(http.ResponseWriter, *http.Request) {
-
-	if c.cfg.Password == nil {
-		return fn
-	}
 	return func(rw http.ResponseWriter, r *http.Request) {
-		hash := r.Header.Get("X-CSAF-PROVIDER-AUTH")
-		if !c.cfg.checkPassword(hash) {
+
+		verify := os.Getenv("SSL_CLIENT_VERIFY")
+		log.Printf("SSL_CLIENT_VERIFY: %s\n", verify)
+
+		switch {
+		case verify == "SUCCESS" && os.Getenv("SSL_CLIENT_I_DN") == c.cfg.Issuer:
```

Comment:
Where is the documentation for this config options, especially the format it accepts.

Reply:
@Fadiabb please add that documentation.